apps/frontend/src/pages/servers/index.vue

```diff
@@ -594,8 +610,8 @@
                 target="_self"
                 class="!bg-highlight-green !font-medium !text-green"
               >
-                Login
```

Comment:
in my opinion the text shouldn't change, it should still say Get started but prompt a login before opening the modal. im guessing this was done because currently there's no direct link to open the modal so the login can't redirect you properly? i imagine the fix for that would just be a hash anchor though

Reply:
As signing in is required to buy a server anyways, that text was implemented there for clarity to users. Although, I agree that 'Get started' seems more appealing than 'Sign in,' so I'll make the text not change

Currently after signing in, it redirects you back to the plans section instead of opening what you clicked. However, I'll see if I can make it so that it automatically launches the modal as well

Reply:
Added query string support to plans in latest commit, so you can do `http://modrinth.com/servers?plan=custom` now, which is also utilised when logging in.

Also removed the 'Sign in' on buttons thing, it just says 'Get started' or 'Out of stock' now

Reply:
\o/